theano/gof/opt.py

```diff
@@ -2448,6 +2601,10 @@ def print_profile(stream, prof, level=0):
                     print(blanc, "merge not implemented for ", o)
             for o, prof in zip(opt.cleanup_optimizers, cleanup_sub_profs[i]):
                 try:
+                    if isinstance(o, MergeOptimizer):
+                        if len(prof) == 8:
+                            prof = prof[:-1]
+                            assert len(prof) == 7
```

Comment:
Remove. I think I fixed the root problem, so this work around isn't needed anymore.